test/oidcc/plug/utils_test.exs

```diff
@@ -1,9 +1,47 @@
+alias Oidcc.Plug.ClientStore
```

Comment:
could also have this alias in each of the test modules if having an alias at the file scope level feels weird